Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -161,6 +161,11 @@
        $ApplicationName = Get-ApplicationNameFromApplicationParameterFile $ApplicationParameterFilePath
     }
 
+    if($ApplicationName -eq '')
```

Comment:
if(!$ApplicationName) would be more consistent with the above check and would handle null and empty string.